search/utils.py

```diff
@@ -65,3 +70,15 @@ def upper_string(self):
     def lower_string(self):
         """ use isoformat for _lower date's string format """
         return self._lower.isoformat()
+
+
+def doc_type_warning_decorator(func):
```

Comment:
It is not a brilliant idea to call decorator as the decorator. It is pretty obvious, isn't it?

Reply:
Thanks